src/main/java/step3/view/InputView.java

```diff
@@ -0,0 +1,27 @@
+package step3.view;
+
+import java.util.Scanner;
+
+public class InputView {
+
+    private static int cars;
+    private static int attemps;
+
+    public static void inputValues() {
+        Scanner scan = new Scanner(System.in);
+
+        System.out.println("자동차 대수는 몇 대인가요?");
+        cars = scan.nextInt();
+
+        System.out.println("시도할 회수는 몇 회 인가요?");
+        attemps = scan.nextInt();
+    }
+
+    public static int getNumberOfCars() {
+        return cars;
+    }
+
+    public static int getAttemps() {
+        return attemps;
+    }
```

Comment:
```suggestion
    public static int getNumberOfCars() {
        return numberOfCars;
    }

    public static int getNumberOfAttemps() {
        return numberOfAttemps;
    }
```

이름이 길더지더라도, 메서드와 관련있는 맴버가 서로 연관되도록 네이밍을 하면 좋을 것 같습니다.

Reply:
이 부분도 앞으로 명심하겠습니다!